### PR TITLE
WFLY-15792 Simplify getTimeZone in CalendarBasedTimeout and remove re…

### PR DESCRIPTION
**Pull request by chengfang**
…dundant null checks

https://issues.redhat.com/browse/WFLY-15792

### CODE REVIEW

**Comment by darranl** [comment]
@tadamski Are you able to add a review to this one?

File: ejb3/src/main/java/org/jboss/as/ejb3/timerservice/schedule/CalendarBasedTimeout.java [hunk 1]
```diff
@@ -539,47 +530,6 @@ private boolean isDayOfMonthWildcard() {
         return this.scheduleExpression.getDayOfMonth().equals("*");
     }
 
-    private void nullCheckScheduleAttributes(ScheduleExpression schedule) {
```
Comment (tadamski) [inline ejb3/src/main/java/org/jboss/as/ejb3/timerservice/schedule/CalendarBasedTimeout.java:542]: @chengfang Are we certain that those are not null? Are all those parts of code reduntant?

Comment (chengfang) [reply ejb3/src/main/java/org/jboss/as/ejb3/timerservice/schedule/CalendarBasedTimeout.java:542]: These 7 fields all have non-null defaults: https://github.com/jakartaee/enterprise-beans/blob/master/api/src/main/java/jakarta/ejb/ScheduleExpression.java#L533-L540

So unless the user intentionally set them to null, they will not be null.

Comment (tadamski) [reply ejb3/src/main/java/org/jboss/as/ejb3/timerservice/schedule/CalendarBasedTimeout.java:542]: ok, thanks for claryfing

**Review by tadamski — APPROVED** [review]

**Pull request merged**